pilot/pkg/kube/inject/inject.go

```diff
@@ -69,6 +69,10 @@ var (
 			"Rewrite HTTP readiness and liveness probes to be redirected to istio-proxy sidecar").Name: alwaysValidFunc,
 		annotations.Register("sidecar.istio.io/proxyImage", "").Name:                           alwaysValidFunc,
```

Comment:
Ran a grep and looks like we use:

`readiness.status.sidecar.istio.io/applicationPorts`
`readiness.status.sidecar.istio.io/failureThreshold`
`readiness.status.sidecar.istio.io/initialDelaySeconds`
`readiness.status.sidecar.istio.io/periodSeconds`
`sidecar.istio.io/bootstrapOverride`
`sidecar.istio.io/controlPlaneAuthPolicy`
`sidecar.istio.io/discoveryAddress`
`sidecar.istio.io/interceptionMode`
`sidecar.istio.io/proxyCPU`
`sidecar.istio.io/proxyImage`
`sidecar.istio.io/proxyMemory`
`sidecar.istio.io/userVolume`
`sidecar.istio.io/userVolumeMount`
`status.sidecar.istio.io/port`
`traffic.sidecar.istio.io/excludeInboundPorts`
`traffic.sidecar.istio.io/excludeOutboundIPRanges`
`traffic.sidecar.istio.io/excludeOutboundPorts`
`traffic.sidecar.istio.io/includeInboundPorts`
`traffic.sidecar.istio.io/includeOutboundIPRanges`
`traffic.sidecar.istio.io/kubevirtInterfaces`


So looks like we are missing proxyCPU, proxyMemory, userVolume, userVolumeMount, controlPlaneAuthPolicy, discoveryAddress. Maybe some more I missed?

Every annotation allowed is supposed to be in the registry, right?

```
cat install/kubernetes/helm/istio/files/injection-template.yaml | rg --only-matching --no-filename --no-heading --no-line-number '`.+?istio.io.+?`' | sort | uniq
```